tests/publiccloud/storage_perf.pm

```diff
@@ -0,0 +1,144 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Use FIO tool to run storage performance test
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+use base "publiccloud::basetest";
```

Comment:
I prefer:
```suggestion
use Mojo::Base 'publiccloud::basetest';
```

Reply:
me too , but do you know what the actual difference between this two ?

Reply:
Its written here: https://mojolicious.org/perldoc/Mojo/Base
So this Mojo::Base is a short variation of some nice and common things like:
```
use strict;
use warnings;
use utf8;
use feature ':5.10';
```

Reply:
done